src/core/fonts.js

```diff
@@ -2883,12 +2823,14 @@ var Font = (function FontClosure() {
         var seac = this.seacMap[charcode];
         fontCharCode = seac.baseFontCharCode;
         accent = {
-          fontChar: String.fromCharCode(seac.accentFontCharCode),
+          fontChar: String.fromCodePoint(seac.accentFontCharCode),
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint#Browser_compatibility this (unsurprisingly) isn't supported by IE11.

Please include the necessary `string/from-code-point` polyfill, using [core-js](https://github.com/zloirock/core-js#ecmascript-string-and-regexp), in the `src/shared/compatibility.js` file.